docs/current/guides/snippets/build-test-publish-php/dagger.php

```diff
@@ -178,7 +178,7 @@ public function runUnitTests($image) {
               withEnvVariable(name: "MARIADB_PASSWORD", value: "t_password") {
                 withEnvVariable(name: "MARIADB_ROOT_PASSWORD", value: "root") {
                   withExposedPort(port: 3306) {
-                    withExec(args: []) {
+                    service() {
```

Comment:
@vito for review

Reply:
Tiny tweak for GraphQL style:

```suggestion
                    service {
```